src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1449,8 +1451,6 @@ void fromNetwork (
     using namespace std::chrono_literals;
     auto constexpr RPC_NOTIFY = 10min;
 
-    auto j = logs.journal ("HTTPClient");
-
     HTTPClient::request (
```

Comment:
I don't have a strong opinion about this, although that change would imply that the caller of HTTPClient::request (or get) is able to set the name of the journal (which just amounts to a log label...), so potentially the caller of these functions could choose to name the journal something other than HTTPClient -- Would we care if they did?